gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartPage.tsx

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
+import NxBinaryDonutChartMinimalExample from './NxBinaryDonutChartMinimalExample';
+import NxBinaryDonutChartNoHoleExample from './NxBinaryDonutChartNoHoleExample';
+import NxBinaryDonutChartLargeHoleExample from './NxBinaryDonutChartLargeHoleExample';
+
+const nxBinaryDonutChartMinimalExampleCode = require('!!raw-loader!./NxBinaryDonutChartMinimalExample').default;
+const nxBinaryDonutChartNoHoleExample = require('!!raw-loader!./NxBinaryDonutChartNoHoleExample').default;
+const nxBinaryDonutChartLargeHoleExample = require('!!raw-loader!./NxBinaryDonutChartLargeHoleExample').default;
+
+const NxBinaryDonutChartPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        <code className="nx-code">NxBinaryDonutChart</code> represents a binary donut chart.
+      </p>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="Minimal Example"
+                        className="nx-binary-donut-chart-examples"
```

Comment:
We don't typically have classNames here but we do often have ids.  Those ids pretty much only important for visual tests.  Which, btw, you are missing

Reply:
> We don't typically have classNames here but we do often have ids. Those ids pretty much only important for visual tests. Which, btw, you are missing

I added one visual test a63e101f8437f7be35a7e2cea5eb4c594c505506 , since basically there are no actions on that component I didn't think it was necessary to do the same for the other percentages, but I can add more if you think it's a good idea

Reply:
> I added one visual test a63e101 , since basically there are no actions on that component I didn't think it was necessary to do the same for the other percentages, but I can add more if you think it's a good idea

You used the same value for each id.  All ids on an HTML page at any given time must be unique.

Additionally.  I think visual tests of varying percentages are necessary for the same reason that examples of varying percentages are necessary: to make sure they look right :)  Similarly, I do also think the example with a 0 inner radius should have a visual test as that's kind of a special case that the browser would be more likely to screw up rendering.

Reply:
> > I added one visual test [a63e101](https://github.com/sonatype/sonatype-react-shared-components/commit/a63e101f8437f7be35a7e2cea5eb4c594c505506) , since basically there are no actions on that component I didn't think it was necessary to do the same for the other percentages, but I can add more if you think it's a good idea
> 
> You used the same value for each id. All ids on an HTML page at any given time must be unique.
> 
> Additionally. I think visual tests of varying percentages are necessary for the same reason that examples of varying percentages are necessary: to make sure they look right :) Similarly, I do also think the example with a 0 inner radius should have a visual test as that's kind of a special case that the browser would be more likely to screw up rendering.

Added b9f8078b190bac0c253d7a7c363ad492ea6a5587